backend/src/routes/TripRouter.ts

```diff
@@ -42,7 +46,8 @@ router.get("/:id", async (req, res) => {
 });
 
 // Create a new trip
-router.post("/", async (req, res) => {
+// router.post("/", async (req, res) => {
+router.post("/", validateData(tripCreateSchema), async (req, res) => {
```

Comment:
I think `validateData` is a nice abstraction, but it doesn't give us the benefit with using typescript. Specifically, line 51 has type `any`.